### ISSUE TITLE
Chunks not saving?

### ISSUE THREAD

**Issue by kkbizz** [NillerMedDild/Enigmatica6#96]
**Modpack version**
0.2.5

**Issue**
It looks like chunks are not saving or at least the chunks with an ocean monument in them.
I've placed around 5 stacks of glass to encase one(making a prismarine/xp farm) and went back to my base to get some more. 
Came back everything was reset. Not a single piece of glass in sight, even the terrain was reset.
The elder guardians were already dead(killed them).

**Logs**
Not applicable. If needed i can upload the world or send screenshots.

**Comment by NillerMedDild** [comment]
Thank you for the report, I will investigate the issue.

**Comment by NillerMedDild** [comment]
Is this on a server or in SP?

**Comment by kkbizz** [comment]
Single player, I first thought it was an issue with the artifact magic mirror but when leaving the chunks and going back without teleporting resulted in the same issue.

**Comment by NillerMedDild** [comment]
May I have your latest.log please?

**Comment by kkbizz** [comment]
[latest.log](https://github.com/NillerMedDild/Enigmatica6/files/5043038/latest.log)
it was to big for pastebin so hope this works :P

**Comment by Kronos967** [comment]
I had the same issue in the Nether.  It appears the backups are not processing.  Here are 2 screenshots.  I completed laying the track formed the portal and lit it.  Saved and quit.  when I came back in, all my work was undone.  Single Player world.  

![2020-08-07_14 20 33](https://user-images.githubusercontent.com/69364158/89677811-166e4c80-d8bc-11ea-9cb4-bb03c70a9fd4.png)
![2020-08-07_14 25 16](https://user-images.githubusercontent.com/69364158/89677821-1a01d380-d8bc-11ea-9509-0ec207ea7efa.png)

Lastlog
[latest.log](https://github.com/NillerMedDild/Enigmatica6/files/5043103/latest.log)
